docs/contribute.rst

```diff
@@ -172,6 +172,12 @@ Helpful commands
     >>> to_checksum_address("0x9c78ee466d6cb57a4d01fd887d2b5dfb2d46288f")
     '0x9C78EE466D6Cb57A4d01Fd887D2b5dFb2D46288f'
 
+Working with the develop branch
+=================================
+
+The big changes to the code all happen in the ``develop`` branch. Those might include changes to the schema both in the user database and the global database. Errors related to partial migrated databases ight manifest as errors in the UI when executing queries or failures to start the app or sign in. For the develop experience instead of the normal ``data`` folder we use another in the same path called ``develop_data`` and contains both users' db and the ``global_data`` folder.
+To avoid loosing information we recomend copying your account from ``data`` to ``develop_data`` along your ``global_data`` folder. If any error related to a half migration appears you can just delete your database at ``develop_data`` and copy it again from ``data`` without loosing any important information.
```

Comment:
```suggestion
To avoid losing information we recomend copying your account from ``data`` to ``develop_data`` each time you pull new changes in develop, especially if you know that any DB schema changes happened.
```